src/protocolsupport/protocol/packet/handler/PSInitialHandler.java

```diff
@@ -218,7 +218,7 @@ protected void processLoginStart() {
 				return;
 			}
 			case PE: {
-				if (isOnlineMode()) {
+				if (true || isOnlineMode()) {
```

Comment:
?

Reply:
I think I made a fucky wucky on my branch, I added this because for some reason authentication wasn't working, let me disable that.

Reply:
Actually, it seems @colinrgodsey already removed my mistake

Reply:
ohhh that was you lol. i thought that was me

Reply:
@MrPowerGamerBR i dont even think i looked what i pulled over from that branch.. just seemed like the most up-to-date code 🤷‍♂️